_topic_maps/_topic_map.yml

```diff
@@ -3645,7 +3645,7 @@ Topics:
   - Name: Getting started with OKD Virtualization
     File: virt-getting-started
     Distros: openshift-origin
-  - Name: Using the virtctl and libguestfs CLI tools
+  - Name: Virtctl and libguestfs
```

Comment:
I'm not seeing the update yet in the preview link. Should `Virtctl` be lower case? I'd imagine the true definition name is in lowercase?

Reply:
For example: https://kubevirt.io/user-guide/operations/virtctl_client_tool/

^^ All instances are lowercase.